### ISSUE TITLE
全数把握見直しに伴い発生事例一覧の公表終了(2022年9月27日公表分より)

### ISSUE THREAD

**Issue by takainou** [code4nagoya/covid19#1455]
## 起こっている問題 / The Problem
陽性者の全数把握見直し(2022/9/26～)に伴い、愛知県新型コロナウイルス感染症対策サイトにて、2022/9/27公表分以降の「発生事例一覧PDF」の公表が終了しました。(日毎の新規陽性者数[年代別/医療機関所在地別]は公表されています)

当サイトでは「発生事例一覧PDF」を元に"新規陽性者数"を算出して、以下グラフ等の表示に利用しており、「2022/9/27以降の新規陽性者数＝0人」として処理されると思われます( #429 )。
- 「新規感染者数」推移グラフ
- 「陽性率・検査実施人数」推移グラフの"陽性率"/"陽性者数"
- 「判断指標と達成状況」の"新規感染者数"/"陽性率"/"70歳以上"
- 「市町村別感染状況」カラム地図

なお「検査陽性者の状況」は全数把握見直し後も以前と同様に公表されており、以下グラフ等への影響は無いと思われます。
- 「入院患者数」推移グラフ
- 「重症者数」推移グラフ
- 「検査陽性者状況の推移」グラフ
- 「検査陽性者の状況」表
- 「判断指標と達成状況」の"入院患者数"/"重症者数"
- 「陽性率・検査実施人数」推移グラフの"検査人数"

## スクリーンショット / Screenshot
![image](https://user-images.githubusercontent.com/63571899/192708554-1f22ac61-deef-40af-bcb7-d4752f477806.png)
![image](https://user-images.githubusercontent.com/63571899/192806076-bd0672c5-b1cb-4043-8987-06ee37a6ad3f.png)

**Comment by mimikun** [comment]
他のグラフについてはよくわからないですが、「新規感染者数」推移グラフについては、[このサイト](https://www.pref.aichi.jp/site/covid19-aichi/index-2.html) の 「新型コロナウイルス感染者の発生について」で判別することはできないでしょうか？

**Comment by takainou** [comment]
>「新型コロナウイルス感染者の発生について」で判別することはできないでしょうか？

そうですね。そちらに毎日公表されている日毎の新規陽性者数[年代別/医療機関所在地別]の2022年9月27日以降公表分[PDF](https://www.pref.aichi.jp/uploaded/attachment/434220.pdf)を読み取れば、可能かと思います(年代別/地域別も)。

「日毎の新規陽性者数」だけであれば、県HPの「検査陽性者の状況」画像をスプレッドシートmain_summaryに入力し「入院患者数」推移グラフ等で利用されており、その中の「累計陽性者数」の前日からの差分を算出する事でも、可能かと思います。
- 「新規感染者数」推移グラフ
- 「陽性率・検査実施人数」推移グラフの"陽性率"/"陽性者数"

なお、新規陽性者数の年代別/地域別は、[既に名古屋市公表分(2月～9/26)と豊橋市公表分(9/6～26)が把握できておらず](https://github.com/code4nagoya/covid19-aichi-tools/issues/99)、9/27以降を取り込んでも、途中のデータが抜ける事となります。
- 「判断指標と達成状況」の"70歳以上"
- 「市町村別感染状況」カラム地図
